src/Blueprint/Blueprint/help/Export-AzBlueprintWithArtifact.md

```diff
@@ -0,0 +1,126 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.Blueprint.dll-Help.xml
+Module Name: Az.Blueprint
+online version: https://docs.microsoft.com/en-us/powershell/module/az.blueprint/export-azblueprintwithartifact
+schema: 2.0.0
+---
+
+# Export-AzBlueprintWithArtifact
+
+## SYNOPSIS
+Export specified blueprint to the specified output location as a JSON file. 
+
+## SYNTAX
+
+```
+Export-AzBlueprintWithArtifact -Blueprint <PSBlueprintBase> -OutputPath <String> [-Version <String>] [-Force]
+ [-DefaultProfile <IAzureContextContainer>] [<CommonParameters>]
+```
+
+## DESCRIPTION
+Export a blueprint with it's artifact and save to disk.
+
+## EXAMPLES
+
+### Example 1
+```powershell
+PS C:\> $bp = Get-AzBlueprint -Name SimpleBlueprint
```

Comment:
Consider removing the Windows specific prompt portion on each line... `PS C:\> `.  Az module is cross-platform capable.